### ISSUE TITLE
Create database context (code-first)

### ISSUE THREAD

**Issue by Almantask** [csinn/Painted-Prosthetics#44]
- [ ] Amputee
- [ ] User
- [ ] ProstheticRequest
- [ ] Artwork
- [ ] Artist
- [ ] Portfolio
- [ ] Donation
- [ ] ArtistRequest
- [ ] Feedback

**Comment by RyanThiele** [comment]
Will this be in a separate library, or in the WebApi library, or somewhere else?

**Comment by Almantask** [comment]
> Will this be in a separate library, or in the WebApi library, or somewhere else?

Yup, seperate. https://github.com/csinn/Painted-Prosthetics/tree/master/Src/Infrastructure/PaintedProsthetics.Persistence

**Comment by RyanThiele** [comment]
Created initial PR #63. It doesn't have to be merged at the moment. It's more of a place holder we can add to before merging.
